src/v3/src/components/LoopbackProbe/LoopbackProbe.tsx

```diff
@@ -180,7 +189,7 @@ const LoopbackProbe: FunctionComponent<{ uischema: LoopbackProbeElement }> = ({
           break;
         } catch (e) {
           // only for unexpected error conditions (e.g. fetch throws an error)
-          Logger.error(`Something unexpected happened while we were checking port ${port}`);
```

Comment:
do we need to make similar changes like V2 to handle max ports reached use case?

Reply:
V2 already handling the max ports reached use case [here](https://github.com/okta/okta-signin-widget/pull/3650/files#diff-5383ac39ca9bdcbf8c11884ac254718b02024b36a4b438c1917e16e444e10ac8R194)
I suppose you are referring to do the same checking in [the https here](https://github.com/okta/okta-signin-widget/pull/3650/files#diff-5383ac39ca9bdcbf8c11884ac254718b02024b36a4b438c1917e16e444e10ac8R176), but I think there is no need, because we always do https first and doubled `maxNumberOfPorts`, so the `countFailedPorts` should not reach to `maxNumberOfPorts`

Reply:
sorry i meant V3 here

Reply:
I think there is no need, the V3 code is much cleaner, it just iterate with each domain+port with async/await, and it didn't check max ports before. But I will ask SIW team to double check

Reply:
Hi @shuowu-okta , would you also take a look at this question? I think we don't need to check the `maxNumberOfPorts` like in V2, but would be better if you can double check